internal/app/app.go

```diff
@@ -72,7 +75,7 @@ func NewClient(ctx context.Context, logger log.Logger) bot.Client {
 // NewIncidentRepository creates a new connection with the database for incidents
 func NewIncidentRepository(ctx context.Context, logger log.Logger) model.IncidentRepository {
 	configuredDatabase := config.Env.Database
-	logger.Info(ctx, fmt.Sprintf(
+	logger.Debug(ctx, fmt.Sprintf(
```

Comment:
I would actually keep this one as Info, since it is just a one-line log, runs only once, in the beginning, and brings a relevant information (the configured incident repository)